File: server/groute/src/main/java/com/ssafy/groute/dto/Route.java
package com.ssafy.groute.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Route {
    private int id;
    private String name;
    private int day;
    private String memo;
    private String isCustom;

    public Route(String name, int day, String memo, String isCustom) {
        this.name = name;
        this.day = day;
        this.memo = memo;
        this.isCustom = isCustom;
    }
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/Route.kt
package com.ssafy.groute.src.dto

data class Route(
    val day: Int,
    val id: Int,
    val isCustom: String,
    val memo: String,
    val name: String,
    var routeDetailList: List<RouteDetail>
)  {
    constructor():this(0,0,"","","", listOf())

}